### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by bacdayhx**
- [x] Have you followed the [contributing guidelines](https://github.com/github/opensource.guide/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you explained what your changes do, and why they add value to the Guides?

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

-----

### CODE REVIEW

**Comment by bacdayhx** [comment]
P

**Comment by bacdayhx** [comment]
Ok

**Review by Saher14 — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -1,4 +1,8 @@
+@@ -1,4 +1,6 @@
 ---
 layout: index
 lang: en
 ---
+---Before you start working on a translation, look through the [open pull requests](https://github.com/github/opensource.guide/pulls) to see if anyone else is already working on one for your language.
+1. [Fork this repository](https://github.com/github/opensource.guide/fork)
+1. [Set up your environment
```
Comment (Saher14) [inline index.html:8]: index.html

**Review by Saher14 — APPROVED** [review]

**Comment by Saher14** [comment]
> * [x]  Have you followed the [contributing guidelines](https://github.com/github/opensource.guide/blob/HEAD/CONTRIBUTING.md)?
> * [x]  Have you explained what your changes do, and why they add value to the Guides?
> 
> **Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

**Pull request closed without merging**